test/check-application

```diff
@@ -22,6 +24,13 @@ registries = ['localhost:5000']
 """
 
 class TestApplication(testlib.MachineCase):
+    def setUp(self):
+        super(TestApplication, self).setUp()
+
+        podman_version = self.machine.execute("rpm -qi podman  | grep Version | cut -d ':' -f2 | cut -c2-")
+        tests_enabled = StrictVersion(podman_version) >= StrictVersion('1.2.0')
+        if not tests_enabled:
+            raise unittest.SkipTest("podman version if not sufficient")
```

Comment:
I don't understand why this is necessary. The package should fail to install if the podman version is not sufficient, i. e. the fedora-29 test should fail during image preparation. I wonder why it doesn't?

Reply:
As discussed on IRC it was the missing epoch in the spec file. Thanks for solving this.